internal/scripts/dev_tools/generate_release_notes.py

```diff
@@ -0,0 +1,129 @@
+#! /usr/bin/env python3
+
+import sys
+import os
+
+from github import Github
+from datetime import datetime, timezone
+
+GITHUB_ORG = 'dtcenter'
+
+CATEGORIES = (
+    'Enhancement',
+    'Bugfix',
+    'New Wrapper',
+    'New Use Case',
+    'Documentation',
+    'Internal',
```

Comment:
```suggestion
    'Task',
```
If you go with the `Internal` -> `Task` change, then this would need to change as well. I agree that `Task` isn't a great name though.

Reply:
`Internal` is a release notes category from previous releases, so I don't think we should change this to `Task`. See comment about Task vs. Internal in the task issue template.